user/TASK/chassis_task/chassis_task.h

```diff
@@ -56,16 +69,21 @@ typedef struct
     const motor_measure_t *motor_feedback;
     
     //Current speed read from motors
-    int16_t pos_read;
+    uint16_t pos_read;
     int16_t speed_read;
     int16_t current_read;
     
     //Target speed set by user/remote control
     int16_t pos_set;
     int16_t speed_set;
+    // TODO: check if these should be unsigned /exist at all
```

Comment:
- "check if these should be unsigned" It seems to work being signed
- "exist at all" - speed_set is used as the chassis speed, independent of any one wheel so probably needed - pos_set is not used so could be deleted

Reply:
Deleted unused.